test/lit/wasm-split/minimized-exports.wast

```diff
@@ -0,0 +1,38 @@
+;; RUN: wasm-split %s --keep-funcs=foo,bar --export-prefix='%' -o1 %t.1.wasm -o2 %t.2.wasm
+;; RUN: wasm-dis %t.1.wasm | filecheck %s --check-prefix PRIMARY
+;; RUN: wasm-dis %t.2.wasm | filecheck %s --check-prefix SECONDARY
+
+;; PRIMARY:      (module
+;; PRIMARY-NEXT:   (type $none_=>_none (func))
+;; PRIMARY-NEXT:   (export "%a" (func $1))
+;; PRIMARY-NEXT:   (export "%b" (func $0))
```

Comment:
I'm missing something... now that this uses the shared minified names generator, that should be emitting valid JS names, but `%a` isn't such?

Reply:
This is using the JS-friendly minified names with a `%` prefix manually specified on the command line.

Reply:
In particular, the `%` prefix is what the Emscripten integration currently uses to differentiate wasm-split exports from "real" exports, so this models how wasm-split would actually be invoked in practice.

Reply:
Oh, I see, I wasn't aware of how it worked on the emscripten side. So it's intentionally not a JS-friendly prefix. (That removes some of the benefit of sharing the JS-friendly minification code here, I suppose, but it's still simpler I think.)